cockatrice/src/settingscache.cpp

```diff
@@ -162,7 +162,7 @@ SettingsCache::SettingsCache()
     notifyAboutUpdates = settings->value("personal/updatenotification", true).toBool();
     lang = settings->value("personal/lang").toString();
     keepalive = settings->value("personal/keepalive", 5).toInt();
-    idlekeepalive = settings->value("personal/idlekeepalive", 36000).toInt();
+    idlekeepalive = settings->value("personal/idlekeepalive", 3600).toInt();
```

Comment:
For time related stuff like this I prefer using something like `60 * 60` to make it totally clear, but this is also fine